jest.setup.js

```diff
@@ -0,0 +1,9 @@
+const { execSync } = require('child_process');
+
+module.exports = function () {
+	console.log('\nSetup Docker for testing');
+	const stdout = execSync(
+		'docker-compose -p docker_app_test -f docker-compose.test.yml up -d',
+	);
+	console.log(stdout.toString('utf-8'));
+};
```

Comment:
Setup Postgres image(docker-compose.test.yml) for integration tests.
`jest.setup.js` and `jest.teardown.js` are executed on start running tests and finish running tests respectively.